python/bin/suricatasc

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/python
-# Copyright(C) 2013 Open Information Security Foundation
+# Copyright(C) 2013-2019 Open Information Security Foundation
 
-# This program is free software; you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, version 2 of the License.
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
```

Comment:
Oops. Missed this the first time around.  We're strictly version 2, we avoid the the later version clause.

Reply:
ah interest, thanks Jason. Updated in new PR.